packages/maskbook/src/utils/shadow-root/ShadowRootMenu.tsx

```diff
@@ -0,0 +1,29 @@
+import { useRef } from 'react'
+import { MenuProps, Menu } from '@material-ui/core'
+import '../../utils/shadow-root/ShadowRootPortal'
+import { PortalShadowRoot } from '../../utils/shadow-root/ShadowRootPortal'
+import { useSheetsRegistryStyles } from './renderInShadowRoot'
+import { ErrorBoundary } from '../../components/shared/ErrorBoundary'
+
+/**
+ * This is the low-level API to rendering a menu.
+ * Please use InjectedMenu if possible.
+ */
+export default function ShadowRootMenu(_props: MenuProps) {
```

Comment:
IIRC we have a `useMenu` handle ShadowRoot correctly.

Reply:
Not working on the content page.